_posts/2016-04-10-docker-inside-docker-and-jenkins-2.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Docker inside docker and overview about Jenkins 2"
+date:   2016-04-1 10:08:27
+categories: docker
+img: /img/docker.png
+tags: docker, oss, jenkins
+summary: "A little overview about Jenkins 2 but the main topic of the article
+is about how run docker inside docker to start a continuous integration system
+inside a container"
+priority: 0.6
+changefreq: yearly
+---
+
+Docker inside docker and Jenkins 2 Jenkins  is one of the most famous
```

Comment:
I think, the Sentence is starting at `Jenkins is` either have a `.`  after 2 or remove it